src/silx/gui/widgets/FrameBrowser.py

```diff
@@ -294,13 +329,17 @@ def __init__(self, parent=None):
         self._playButton.setToolTip("Display movie with frames")
         self._playButton.setIcon(icons.getQIcon("camera"))
         self._playButton.setIconSize(iconSize)
+        self._playButton.setCheckable(True)
```

Comment:
Use button checkstate rather than changing the icon (or the a real stop action should be used)